### PR TITLE
Ai tabbed interface

### PR DESCRIPTION
**Pull request by aizatibraimova**

### CODE REVIEW

File: app/views/users/_follow.html.erb
Comment (jniewiel) [inline app/views/users/_follow.html.erb:1]: Great job on creating a seperate render for your follows!

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+      <li class="nav-item">
+        <%= link_to "Feed", feed_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Followers", followers_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link" %>
+      </li>
```
Comment (jniewiel) [inline app/views/users/show.html.erb:22]: Nice work on implementing your paths to the User's other nav_items!

**Review by jniewiel — APPROVED** [review]
Great job on the tabbed interface!

File: app/views/users/feed.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    <ul class="nav nav-pills nav-justified">
+      <li class="nav-item">
+        <%= link_to "Posts", user_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Liked photos", liked_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Feed", feed_path(@user.username), class: "nav-link active" %>
```
Comment (aborov) [inline app/views/users/feed.html.erb:15]: I love that you use Bootstrap `active` class to let a user see where they current are

File: app/views/users/liked.html.erb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+      <li class="nav-item">
+        <%= link_to "Feed", feed_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Followers", followers_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link" %>
+      </li>
```
Comment (aborov) [inline app/views/users/liked.html.erb:22]: Great job using helper method `link_to` to make your navigation links reusable!

File: config/routes.rb [hunk 4]
```diff
@@ -1,13 +1,21 @@
...
-  resources :follow_requests
   resources :comments
+  resources :follow_requests
+  resources :likes
   resources :photos
 
+  get ":username/liked" => "users#liked", as: :liked
+  get ":username/feed" => "users#feed", as: :feed
+  get ":username/followers" => "users#followers", as: :followers
+  get ":username/following" => "users#following", as: :following
```
Comment (aborov) [inline config/routes.rb:14]: Nice job defining the new user-specific routes!

**Review by aborov — APPROVED** [review]
Great job Aizat!

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, Aizat!

In your "users" view folder you could also create a partial for all the tab info and use the partial in the feed, followers, following, liked and show view files. 

You can now merge these changes back to main.

**Comment by CarlosCastanedaDev** [comment]
Please submit this PR link in Canvas so I can grade it and you can move forward.

**Pull request merged**